Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -44,6 +44,16 @@ function handleError(e, isFatal) {
   }
 }
 
+function polyfillGlobal(name, newValue) {
+  var descriptor = Object.getOwnPropertyDescriptor(GLOBAL, name);
+
+  if (typeof GLOBAL[name] !== 'undefined') {
```

Comment:
can you write `GLOBAL[name] !== undefined` or `typeof name !== 'undefined'` but doing typeof + GLOBAL[] is not useful